### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by minuscorp**
The aim of the PR is to update the documentation in order to ease the task of deploying the fastlane.ci in early development stages, as it's very hard to do without help for now! :rocket:

### CODE REVIEW

**Comment by minuscorp** [comment]
Take a review! This should ease things 🚀

**Comment by KrauseFx** [comment]
Is it ready for review and merge? If so, don't forget to remove the `WIP` 👍

File: README.md [hunk 1]
```diff
@@ -91,6 +91,73 @@ export INITIAL_CLONE_EMAIL="email@user.com"
...
+
+```json
+[
+    {
+      "id": "ee75eb27-9246-43c1-af5a-a8d33f8a963f",
+      "email": "your-name@gmail.com",
+      "password_hash": "some password hash that needs to be created",
+      "provider_credentials": [
+        {
+          "email": "minuscorp@gmail.com",
```
Comment (taquitos) [inline README.md]: best email

Comment (minuscorp) [reply README.md]: Whoooops

Comment (taquitos) [reply README.md]: Hahaha, it's ok if you don't mind your email being in public. Mine is all over the place.

**Pull request merged**